src/popup/components/DropDownMenu/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Menu, MenuItem, Button, withStyles } from '@material-ui/core';
+import { ArrowDropDown } from '@material-ui/icons';
+import PropTypes from 'prop-types';
+
+import styles from './styles';
+
+@withStyles(styles, { withTheme: true })
+export default class DropDownMenu extends React.Component {
```

Comment:
I meant this to be a generic component. I was thinking that it might be useful if our settings button which currently just has logout eventually had other selection choices, or if we had other use cases for a drop-down menu